indico/modules/events/abstracts/controllers/reviewing.py

```diff
@@ -41,35 +42,15 @@
 from indico.util.i18n import _
 from indico.web.flask.templating import get_template_module
 from indico.web.util import jsonify_data
-from MaKaC.webinterface.rh.conferenceDisplay import RHConferenceBaseDisplay
-
-
-class RHAbstractsBase(RHConferenceBaseDisplay):
-    CSRF_ENABLED = True
-    EVENT_FEATURE = 'abstracts'
-
-
-class RHAbstractReviewBase(AbstractMixin, RHAbstractsBase):
-    normalize_url_spec = {
-        'locators': {
-            lambda self: self.abstract
-        }
-    }
-
-    def _checkProtection(self):
-        RHAbstractsBase._checkProtection(self)
-        AbstractMixin._checkProtection(self)
-
-    def _checkParams(self, params):
-        RHAbstractsBase._checkParams(self, params)
-        AbstractMixin._checkParams(self)
-        self.management = request.view_args.get('management', False)
 
 
 class RHListOtherAbstracts(RHAbstractsBase):
     """AJAX endpoint that lists all abstracts in the event (dict representation)."""
 
-    CSRF_ENABLED = True
+    def _checkProtection(self):
```

Comment:
I think we want a slightly less permissive protection check here. Do we really want to make a list of all abstracts (which can be perceived as confidential information) to all users that can access the event?